stylesheets/user.css

```diff
@@ -0,0 +1,71 @@
+/*body {
```

Comment:
Any reason for committing a fully commented out file?

Reply:
No I was supposed to delete this. Sorry